actionpack/test/controller/request_forgery_protection_test.rb

```diff
@@ -886,9 +903,10 @@ def test_rejects_token_for_incorrect_path
 
     # This is required because PATH_INFO isn't reset between requests.
     @request.env["PATH_INFO"] = "/per_form_tokens/post_two"
-    assert_raises(ActionController::InvalidAuthenticityToken) do
+    exception = assert_raises(ActionController::InvalidAuthenticityToken) do
```

Comment:
I think we don't need to assert the exception message for all tests.
These last three can be reverted I think.
Than this looks good to me!

Reply:
Yep! Changed